crates/control/src/config.rs

```diff
@@ -4,6 +4,7 @@ use serde::Deserialize;
 pub mod app_env;
 
 pub use app_env::app_env;
+use tracing::info;
 
 #[derive(Debug, Deserialize)]
 pub struct Settings {
```

Comment:
Looking at how this is evolving, I'm wondering if it might be worth trying to use `clap` for these. It supports using env variables, and that would mean that everything could be defined in the `.env` files, while also allowing you to override those by passing CLI arguments. It also gives a nice consistent way of getting documentation for all the options using `--help`, and would allow for seamless integration with the rest of `flowctl-rs` (we could add generic support for .env files in `flowctl-rs`). WDYT?

Reply:
Yeah, it has really gotten messier than I'd hoped. The `config` crate offers a lot of the functionality I'd want, but `sqlx` doesn't make using it easy. I was thinking that `flowctl-rs` would have a flag for pointing to a config file/directory to avoid needing to make dozens of flags.

Maybe we can chat about this more on a VC?